### ISSUE TITLE
The two pairs of Dark Rune in Thorim's entryway do not patrol.

### ISSUE THREAD

**Issue by heyitsbench** [azerothcore/azerothcore-wotlk#15402]
### Current Behaviour

The two pairs of Dark Rune dudes in Thorim's entry corridor do not move.

### Expected Blizzlike Behaviour

The pair closer to the Observation Ring should be patrolling through the straight section of the hall, and the further pair should patrol along that 45 degree section.

### Source

Wrath Classic.

### Steps to reproduce the problem

1. Enter Ulduar.
2. Clear your way to the Observation Ring.
3. "Observe" (haha do you get it cause observation ring) the two pairs of Dark Rune trash not moving.

### AC rev. hash/commit

a1c644fd3971085d25d16700384110feda6bfdab

### Operating system

Windows 10 x64

### Custom changes or Modules

[mod-ptr-template](https://github.com/heyitsbench/mod-ptr-template)

**Comment by avarishd** [comment]
Source for both packs patrolling: https://youtu.be/G2EJhb7wCec?t=1685

.go xyz 2084 -110 412 603

**Comment by heyitsbench** [comment]
Here's what the worldserver has to say about these packs.
```
WaypointMovementGenerator::LoadPath: creature Dark Rune Thunderer (GUID Full: 0xf1300083da0005c7 Type: Creature Entry: 33754  Low: 1479) doesn't have waypoint path id: 0
WaypointMovementGenerator::LoadPath: creature Dark Rune Ravager (GUID Full: 0xf1300083db000656 Type: Creature Entry: 33755  Low: 1622) doesn't have waypoint path id: 0
WaypointMovementGenerator::LoadPath: creature Dark Rune Thunderer (GUID Full: 0xf1300083da000657 Type: Creature Entry: 33754  Low: 1623) doesn't have waypoint path id: 0
WaypointMovementGenerator::LoadPath: creature Dark Rune Ravager (GUID Full: 0xf1300083db000658 Type: Creature Entry: 33755  Low: 1624) doesn't have waypoint path id: 0
```

**Comment by elthehablo** [comment]
> 

yup that seems to be the issue. no `path_id` configured for these mobs yet the `waypoint_data` does exist and they are marked to have waypoints, hence why there errors pop up
